src/components/Avatar.jsx

```diff
@@ -12,10 +12,10 @@ const makeStyle = ({ faded, color }) => {
     : { backgroundColor: color }
 }
 
-const Avatar = ({ icon, color, faded }) => {
+const Avatar = ({ icon, color, faded, ghost }) => {
   const style = useMemo(() => makeStyle({ faded, color }), [color, faded])
 
-  return <UiAvatar style={style} icon={icon} size={32} />
+  return <UiAvatar style={style} icon={icon} size={32} ghost={ghost} />
```

Comment:
nit: let's add a test check our avatar provides correctly ghost props to UiAvatar

Reply:
Les tests sur les composants viendront 👍